source/assets/stylesheets/vendor/slick-carousel/slick-theme.scss

```diff
@@ -0,0 +1,235 @@
+@charset "UTF-8";
+
+// Default Variables
+
+// Slick icon entity codes outputs the following
+// "\2190" outputs ascii character "←"
+// "\2192" outputs ascii character "→"
+// "\2022" outputs ascii character "•"
+
+// $slick-font-path: "node_modules/slick-carousel/slick/fonts/" !default;
+// $slick-font-family: "slick" !default;
+//$slick-loader-path: "./" !default;
+$slick-arrow-color: $gray-900 !default;
+$slick-dot-color: $primary !default;
+$slick-dot-color-active: $slick-dot-color !default;
+$slick-prev-character: "\2190" !default;
+$slick-next-character: "\2192" !default;
+$slick-dot-character: "\2022" !default;
+$slick-dot-size: $font-size-base !default;
+$slick-opacity-default: 1 !default;
+$slick-opacity-on-hover: 1 !default;
+$slick-opacity-not-active: 0.75 !default;
+
+// @function slick-image-url($url) {
+//     @if function-exists(image-url) {
+//         @return image-url($url);
+//     }
+//     @else {
+//         @return url($slick-loader-path + $url);
+//     }
+// }
+// 
+// @function slick-font-url($url) {
+//     @if function-exists(font-url) {
+//         @return font-url($url);
+//     }
+//     @else {
+//         @return url($slick-font-path + $url);
+//     }
+// }
+
+/* Slider */
+
+// .slick-list {
+//     .slick-loading & {
+//         background: #fff slick-image-url("ajax-loader.gif") center center no-repeat;
+//     }
+// }
+
+/* Icons */
+// @if $slick-font-family == "slick" {
+//     @font-face {
+//         font-family: "slick";
+//         src: slick-font-url("slick.eot");
+//         src: slick-font-url("slick.eot?#iefix") format("embedded-opentype"), slick-font-url("slick.woff") format("woff"), slick-font-url("slick.ttf") format("truetype"), slick-font-url("slick.svg#slick") format("svg");
+//         font-weight: normal;
+//         font-style: normal;
+//     }
+// }
```

Comment:
Even here, if we are not using these slick styles, it may be better removing them. And also indent code with 2 spaces.